ads/google/a4a/google-data-reporter.js

```diff
@@ -103,7 +103,7 @@ function isInReportableBranch(ampElement, namespace) {
  * @param {string} namespace
  * @param {number|string} slotId A unique numeric identifier in the page for
  *    the given element's slot.
- * @return {!GoogleAdLifecycleReporter|!BaseLifecycleReporter}
+ * @return {!./performance.BaseLifecycleReporter}
  */
 export function getLifecycleReporter(ampElement, namespace, slotId) {
```

Comment:
Ah, good catch.  We used to, but with the current changes we no longer do.  Marking it `@visibleForTesting`.